src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat ;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star--active {
```

Comment:
The modifier should be used, to apply styles. And the user should be able to change the modifier dynamically in the HTML and see the style change interactively